GetStream/stream-chat-android
# [4758] Remove Old Image Gallery Remnants

### 🎯 Goal

closes #4758

### 🛠 Implementation details

Removed remnants of the old image gallery

### 🧪 Testing

The testing section is here only to make sure nothing that was actually used was deleted. Everything should have stayed the same.

1. Open the Compose app 
2. Upload a few image and video attachments and send them
3. Open them inside the gallery, make sure everything works correctly

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![gif](https://media2.giphy.com/media/30pxUsK4pnIG9BgKgw/giphy.gif?cid=ecf05e47jppd7j1uuhnx6293wpmzgyk8j6omx3d2kxzqr8rf&rid=giphy.gif&ct=g)
